packages/ketcher-react/src/script/ui/views/modal/components/toolbox/Bond/Bond.module.less

```diff
@@ -35,4 +35,37 @@
       margin-bottom: 4px
     }
   }
+
+  [aria-haspopup="listbox"][aria-disabled="true"]{
+    cursor: not-allowed!important;
+    border: 1px solid @color-border-input!important;
+    background-color: @color-grey-3!important;
+  }
+}
+
+.customQueryWrapper{
+  label{
```

Comment:
Spaces are missed here and below as well

Reply:
done